cime_config/config_headers.xml

```diff
@@ -28,6 +28,12 @@
     </header>
   </file>
 
+  <file name ="env_mach_specific.xml">
+    <header>
+    These variables should not be changed.
```

Comment:
Let's change this message - How about "These variables control the machine dependent environment including the paths to compilers and libraries external to cime such as netcdf, environment variables for use in the running job should also be set here."